behandlingsprosess/src/main/java/no/nav/foreldrepenger/behandling/steg/kompletthet/VurderKompletthetSteg.java

```diff
@@ -57,6 +61,9 @@ public VurderKompletthetSteg(Kompletthetsjekker kompletthetsjekker, BehandlingRe
             return BehandleStegResultat.utførtUtenAksjonspunkter();
         }
 
+        // Bestill inntektsmeldingforespørsler ved behov
```

Comment:
Kanskje en if !Engangsstønad && !behandling.haraksjonspunkt(7003 -  AUTO_VENTER_PÅ_KOMPLETT_SØKNAD)

Se også på koden for skal + kanPassereKompletthet - om vi bør lage forespørsel i disse tilfellene. En ting er å ikke sende brev til bruker her, men noe til arbeidsgiver kanskje?